pkg/jx/cmd/delete_team.go

```diff
@@ -195,4 +194,11 @@ func (o *DeleteTeamOptions) changeNamespace(ns string) {
 	if err != nil {
 		log.Warnf("Failed to set context to namespace %s: %s", ns, err)
 	}
+
+	//Reset all the cached clients & namespace values when switching so that they can be properly recalculated for
+	//the new namespace.
```

Comment:
I would move this in a helper method in `common.go` such that it can be reused.

Reply:
Yeah, I wondered the same after pushing it - moved into CommonOptions so that everyone can switch NS easily.